course_content/iris_course/4.Cube_Plotting.ipynb

```diff
@@ -148,8 +151,103 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "qplt.contourf(cube[:, 0, :])\n",
-    "plt.show()"
+    "#\n",
+    "# edit space for user code ...\n",
+    "#"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# SAMPLE SOLUTION\n",
+    "# %load solutions/iris_exercise_4.1b"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-warning\">\n",
+    "    <b><font color=\"brown\">Suggested Activity: </font></b>\n",
+    "    How can you extract a 2-dimensional section of this data, to make a useful contour plot?\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#\n",
+    "# edit space for user code ...\n",
+    "#"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# SAMPLE SOLUTION\n",
+    "# %load solutions/iris_exercise_4.1c"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "----\n",
+    "\n",
+    "A useful alternative to contouring is to make a colour 'blockplot', which colours in each datapoint rather than drawing contours.  This works well where contours would be too dense and complicated, or if you need to look at every point in the data.  \n",
+    "In matplotlib, the <a href=\"https://matplotlib.org/api/_as_gen/matplotlib.pyplot.pcolormesh.html\"><b><font face=\"courier\">plt.pcolormesh</font></b></a> method does this."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-warning\">\n",
+    "    <b><font color=\"brown\">Suggested Activity: </font></b>\n",
+    "    <p>Plot the Iris equivalent of the colour blockplot method \n",
+    "      <a href=\"https://matplotlib.org/api/_as_gen/matplotlib.pyplot.pcolormesh.html\">matplotlib.pyplot.pcolormesh</a>    for the first timestep of the 'airtemps' data, i.e. <b><font face=\"courier\" color=\"black\">airtemps[0]</font></b>.</p> \n",
+    "    <p>Zoom in on a small region so you can see the individual data points.</p>\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#\n",
+    "# edit space for user code ...\n",
+    "#"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# SAMPLE SOLUTION\n",
+    "# %load solutions/iris_exercise_4.1d"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "----\n",
+    "\n",
+    "Almost all the Iris plot methods have both `iplt` and `qplt` versions.  \n",
+    "Also, most of these have the same names as similar methods in `matpltloib.pyplot`."
```

Comment:
Could have links to the documentation for [iris.plot](https://scitools.org.uk/iris/docs/latest/iris/iris/plot.html) and iris.quickplot to encourage them to have a look at what is available.

Reply:
:+1: will do